test/unit/player.test.js

```diff
@@ -603,95 +621,91 @@ test('player should handle different error types', function(){
 });
 
 test('Data attributes on the video element should persist in the new wrapper element', function() {
-  var dataId, tag, player;
+  const dataId = 123;
 
-  dataId = 123;
+  const tag = TestHelpers.makeTag();
 
-  tag = TestHelpers.makeTag();
   tag.setAttribute('data-id', dataId);
 
-  player = TestHelpers.makePlayer({}, tag);
+  const player = TestHelpers.makePlayer({}, tag);
 
   equal(player.el().getAttribute('data-id'), dataId, 'data-id should be available on the new player element after creation');
 });
 
-test('should restore attributes from the original video tag when creating a new element', function(){
-  var tag, html5Mock, el;
-
+test('should restore attributes from the original video tag when creating a new element', function() {
   // simulate attributes stored from the original tag
-  tag = Dom.createEl('video');
+  const tag = Dom.createEl('video');
+
   tag.setAttribute('preload', 'auto');
   tag.setAttribute('autoplay', '');
   tag.setAttribute('webkit-playsinline', '');
 
-  html5Mock = { options_: { tag: tag } };
+  const html5Mock = { options_: {tag} };
 
   // set options that should override tag attributes
   html5Mock.options_.preload = 'none';
 
   // create the element
-  el = Html5.prototype.createEl.call(html5Mock);
+  const el = Html5.prototype.createEl.call(html5Mock);
 
   equal(el.getAttribute('preload'), 'none', 'attribute was successful overridden by an option');
   equal(el.getAttribute('autoplay'), '', 'autoplay attribute was set properly');
   equal(el.getAttribute('webkit-playsinline'), '', 'webkit-playsinline attribute was set properly');
 });
 
 test('should honor default inactivity timeout', function() {
-    var player;
-    var clock = sinon.useFakeTimers();
+  const clock = sinon.useFakeTimers();
 
     // default timeout is 2000ms
-    player = TestHelpers.makePlayer({});
+  const player = TestHelpers.makePlayer({});
 
-    equal(player.userActive(), true, 'User is active on creation');
-    clock.tick(1800);
-    equal(player.userActive(), true, 'User is still active');
-    clock.tick(500);
-    equal(player.userActive(), false, 'User is inactive after timeout expired');
+  equal(player.userActive(), true, 'User is active on creation');
+  clock.tick(1800);
+  equal(player.userActive(), true, 'User is still active');
+  clock.tick(500);
+  equal(player.userActive(), false, 'User is inactive after timeout expired');
 
-    clock.restore();
+  clock.restore();
 });
 
 test('should honor configured inactivity timeout', function() {
-    var player;
-    var clock = sinon.useFakeTimers();
+  const clock = sinon.useFakeTimers();
 
     // default timeout is 2000ms, set to shorter 200ms
```

Comment:
the comments in these couple of tests are no longer aligned properly to the code.